### ISSUE TITLE
ccby.pdf not found

### ISSUE THREAD

**Issue by stefan11** [langsci/langscibook#111]
If texlive 2020 is installed with all packages as eg in all Mac installations, loading of the classes fails. I did not notice this before since I had ./langsci// in my TEXINPUTS.

This is what a grep for langsci reveals:
```
grep langsci check-germanic.log
L3 programming layer <2020-03-06> (./langsci/langscibook.cls
LaTeX Warning: You have requested document class `./langsci/langscibook',
               but the document class provides `langscibook'.
Document Class: langscibook 2020/02/12 Language Science Press
Warnings are disabled in langscibook.cls for underfull boxes, hyperref, tocstyl
(/usr/local/texlive/2020/texmf-dist/tex/xelatex/langsci/langsci-basic.sty)
(/usr/local/texlive/2020/texmf-dist/tex/xelatex/langsci/langsci-series.def)
Package biblatex Info: ... file 'langsci.dbx' not found.
Package biblatex Info: Trying to load bibliography style 'langsci-unified'...
Package biblatex Info: ... file 'langsci-unified.bbx' found.
(/Users/stefan/Library/texmf/tex/latex/unified-biblatex/langsci-unified.bbx
Package biblatex Info: Trying to load citation style 'langsci-unified'...
Package biblatex Info: ... file 'langsci-unified.cbx' found.
(/Users/stefan/Library/texmf/tex/latex/unified-biblatex/langsci-unified.cbx
(./styles/oneline.sty) (./langsci/langsci-lgr.sty) (./styles/mycommands.sty)
(./langsci/langsci-gb4e.sty
LaTeX Warning: You have requested package `./langsci/langsci-gb4e',
               but the package provides `langsci-gb4e'.
Package: langsci-gb4e 2020/01/01
) (./langsci/langsci-forest-setup.sty
) (./styles/additional-langsci-index-shortcuts.sty)
(./langsci/langsci-tbls.sty
```
The result is that compilation fails:
```
(/usr/local/texlive/2020/texmf-dist/tex/latex/microtype/mt-msb.cfg)

LaTeX Warning: File `ccby.pdf' not found on input line 25.

! Unable to load picture or PDF file 'ccby.pdf'.
<to be read again> 
                   }
l.25 \maketitle
               
? 
```

The following helps assuming a csh:
```
setenv TEXINPUTS ./langsci//:${TEXINPUTS}
```

**Comment by stefan11** [comment]
I removed the langsci installation to not load this stuff but then latex does not find langsci-basic since it is in ./langsci and not in the standard texpath.

\usepackage{langsci-basic}

**Comment by stefan11** [comment]
OK. Here is what is going on:
```latex
\newcommand{\ccby}{CC-BY}
\newcommand{\ccbynd}{CC-BY-ND}
\ifx\lsCopyright\ccby
Published under the Creative Commons Attribution 4.0 Licence (CC BY 4.0):
http://creativecommons.org/licenses/by/4.0/ \includegraphics[height=.75em]{ccby.pdf}
\else
\ifx\lsCopyright\ccbynd
Published under the Creative Commons Attribution-NoDerivatives 4.0 Licence (CC BY-ND 4.0):
http://creativecommons.org/licenses/by-nd/4.0/ \includegraphics[height=.75em]{ccbynd.pdf}
\else
Published under the Creative Commons Attribution-ShareAlike 4.0 Licence (CC BY-SA 4.0):
http://creativecommons.org/licenses/by-sa/4.0/ \includegraphics[height=.75em]{ccbysa.pdf}
\fi
\fi
```
The pdfs are in a subdirectory named `unchanged`. Without the path specification LaTeX does not find the graphics. I guess if everything has to be flat, you have to move these files up into the main directory of the langsci class.

**Comment by stefan11** [comment]
Copying ccby.pdf into the project directory fixes this. I have learned that this is the normal setup now.
